frontend/src/app/cookie-law-banner/cookie-law-banner.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { CookieService } from 'ngx-cookie';
+
+@Component({
+  selector: 'volontulo-cookie-law-banner',
+  templateUrl: './cookie-law-banner.component.html',
+  styleUrls: ['./cookie-law-banner.component.css']
+})
+export class CookieLawBannerComponent implements OnInit {
+
+  constructor(private _cookieService: CookieService) {
+  }
+
+  ngOnInit() {
+    this.hideCookieBanner(!!this._cookieService.get('cookielaw_accepted'));
+  }
+
+  acceptCookieLaw() {
+    const yearFromNow = new Date().setFullYear(new Date().getFullYear() + 1);
+    this._cookieService.put('cookielaw_accepted', '1', {
+      expires: new Date(yearFromNow)
+    });
+    this.hideCookieBanner(true);
+  }
+
+  private hideCookieBanner(shouldHide: boolean) {
+    document.getElementById('cookielaw_banner').style.display = shouldHide ? 'none' : 'visible';
```

Comment:
will not it be better to use a class instead of changing style directrly?

Reply:
yeah probably